### DISCUSSION TITLE
CTkToolTip & CTkTable

### DISCUSSION THREAD

**Discussion by avalon60** [Akascape/CTkTable#15] [Q&A] [answered]
Hi @Akascape. Is there a way that I can attach a tool-tip to a to a specific column of buttons - or a single button, if there is no way to do it for a column?

**Comment by Akascape** [comment, marked as answer]
@avalon60 
This is the instance of each table cell/button: `table.frame[row, column]`. So simply attach the tooltip to that cell.

**Comment by avalon60** [comment]
Thanks - that worked!
